packages/components/src/__future__/Tabs/_docs/Tabs.mdx

```diff
@@ -0,0 +1,50 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KAIOInstallation } from "~storybook/components"
+import * as TabsStories from "./Tabs.stories"
+
+<Meta of={TabsStories} />
+
+# Tabs
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Tabs"
+  figma="https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-UI-Kit%3A-Heart?type=design&node-id=1929%3A28886&mode=design&t=AGMmnoJia9RscurE-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3081929117/Tabs"
+
+/>
+
+<KAIOInstallation
+  exportNames={["Tabs", "TabList", "Tab", "TabPanel"]}
+  isFuture
+/>
+
+## Overview
+
+<Canvas of={TabsStories.Playground} />
+<Controls of={TabsStories.Playground} />
+
+## Migration from current KAIO Tabs
```

Comment:
Maybe make the migration path a separate file (like what we did with future Tag) :)